index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>COLORANDOM</title>
+    <link href="styles.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <section class="color-random">
+      <article class="title-box">
+        <h1>COLORANDOM</h1>
+      </article>
+      <article class="color-boxes">
+        <div class="box-1 box-format">
+          <div class="color-1 color-format">
+          </div>
+          <div class="hex-and-lock">
+            <h2 class="hex-code">#hex</h2>
+            <img src="assets/unlock-small.png" alt="unlocked lock">
+          </div>
+        </div>
+        <div class="box-2 box-format">
+          <div class="color-2 color-format">
+          </div>
+          <div class="hex-and-lock">
+            <h2 class="hex-code">#hex</h2>
+            <img src="assets/unlock-small.png" alt="unlocked lock">
+          </div>
+        </div>
+        <div class="box-3 box-format">
+          <div class="color-3 color-format">
+          </div>
+          <div class="hex-and-lock">
+            <h2 class="hex-code">#hex</h2>
+            <img src="assets/unlock-small.png" alt="unlocked lock">
+          </div>
+        </div>
+        <div class="box-4 box-format">
+          <div class="color-4 color-format">
+          </div>
+          <div class="hex-and-lock">
+            <h2 class="hex-code">#hex</h2>
+            <img src="assets/unlock-small.png" alt="unlocked lock">
+          </div>
+        </div>
+        <div class="box-5 box-format">
+          <div class="color-5 color-format">
+          </div>
+          <div class="hex-and-lock">
+            <h2 class="hex-code">#hex</h2>
+            <img src="assets/unlock-small.png" alt="unlocked lock">
+          </div>
+        </div>
+      </article>
```

Comment:
These look great so far. Glad we added the classes that apply to all color box display divs